src/components/Quotes.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+import React, { useState, useEffect, useRef } from "react";
+import "./Quotes.css"
+
+const Quote = () => {
+  const [quotes, setQuotes] = useState("");
```

Comment:
- [ ] Please make sure to cover the `Loading` and `Error` states of the app. As the project requirements dictate, just like you set the state of the `quotes`, please do so with the `loading`, and `error`, then write the appropriate conditions to them in the fetch method.